tools/TestModule.ps1

```diff
@@ -35,6 +36,6 @@ $PesterConfiguration.TestResult.Enabled = $true
 $PesterConfiguration.TestResult.OutputPath = (Join-Path $ModuleTestsPath "$moduleName-TestResults.xml")
 
 $TestResults = Invoke-Pester -Configuration $PesterConfiguration
-If ($TestResults.FailedCount -gt 0) { Write-Error "$($TestResults.FailedCount) tests failed." }
```

Comment:
@timayabi2020, why are we changing this to 5? We shouldn't have filing tests.

Reply:
Thank you for flagging this. It was an error from my side. This has been changed